scripts/wiki/ddl.sql

```diff
@@ -0,0 +1,55 @@
+-- DDL for parsed Wiki data.
+
+CREATE TABLE entities (
+    id TEXT PRIMARY KEY NOT NULL,
+    -- This could be normalized. Not worth it at the moment though, since they aren't used.
```

Comment:
What are 'they'?

Reply:
Added in next commit. Please close this conversation if you think the comment is sufficient.